docs/releases/2.7.0.rst

```diff
@@ -0,0 +1,235 @@
+=====================================
+Welcome to the new Pootle 2.7.0 alpha
+=====================================
+
+*Not yet released*
+
+This is the first release of Pootle that combines the work of Evernote and
+Translate.
+
+Changes in Requirements
+=======================
+- Django >= 1.6.10 < 1.7
+- `Translate Toolkit <http://toolkit.translatehouse.org/download.html>`_ >=
+  1.12.0
+- Python >= 2.6
+- Redis - latest stable release.
+- Raven 5.0.0 or higher is now recommended for Sentry support on production
```

Comment:
Note that this is not a hard dep, just an aid for deployments to track their health.